plugins/janus_streaming.c

```diff
@@ -1066,11 +1083,15 @@ typedef struct janus_streaming_rtp_source {
 	janus_streaming_buffer *curldata;
 	char *rtsp_url;
 	char *rtsp_username, *rtsp_password;
-	int ka_timeout;
+	gint64 ka_timeout;
 	char *rtsp_ahost, *rtsp_vhost;
 	gboolean reconnecting;
 	gint64 reconnect_timer;
 	janus_mutex rtsp_mutex;
+	gint64 reconnect_delay; /* Only relevant for RTSP mountpoints */
+	gint64 session_timeout; /* Only relevant for RTSP mountpoints */
+	int rtsp_timeout; /* Only relevant for RTSP mountpoints */
+	int rtsp_conn_timeout; /* Only relevant for RTSP mountpoints */
```

Comment:
I think you can remove the `/* Only relevant for RTSP mountpoints */` comments, they already are in the RTSP only section. Can you also move them before the definition of `rtsp_mutex`? (just for code style, since typically the mutex is the last thing we define in a group)

Reply:
Of course. :) I didn't know about that rule, I moved the mutex to the last line in that section.